packages/mdc-menu/simple/foundation.js

```diff
@@ -37,6 +37,7 @@ export default class MDCSimpleMenuFoundation extends MDCFoundation {
       removeClass: (/* className: string */) => {},
       hasClass: (/* className: string */) => {},
       hasNecessaryDom: () => /* boolean */ false,
+      eventTargetHasClass: (/* target: EventTarget, className: string */) => false,
```

Comment:
IMHO, we should make the method name more generic and straightforward about its intention. I feel `elementHasClass: (/*element: DOMElement, className: string*/` sounds more clear when I implement adapter method because the method is opinion-less about whether it is a event target. And all event target are DOMElement based on [MDN](https://dom.spec.whatwg.org/#dom-event-target). WDYT?